3rdparty/compiler-rt/CMakeLists.txt

```diff
@@ -0,0 +1,202 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+check_submodule_not_empty(../libgcov/compiler-rt)
```

Comment:
The compiler-rt is currently from llvm 7. While the compiler-rt is relative stable, do you see any potential issue if we need to update this in the future? One of the pain point is the LLVM does maintain each library as separated git repo anymore. Not sure if we can to use the approach as musl/mbedtls --- have a clone of the compiler-rt (sub-directory from LLVM repo) under the openenclave organization.

Reply:
compiler-rt is relatively stable. However, LLVM-7's compiler-rt which we use might be quite old at this point.

Since we don't anticipate making changes to compiler-rt, and to avoid the maintenance overhead of having to maintain a clone of the compiler-rt sub-directory, how about we use the published sources from the LLVM release:

https://github.com/llvm/llvm-project/releases/tag/llvmorg-10.0.1

Sources of major components (llvm, clang, compiler-rt, libunwind etc) are published in separated archives too.
compiler-rt-10.0.1.src.tar.xz is only 2MB.

I've made change to download, verify and use the compiler-rt source archive.
What do you think?

Reply:
Sounds good to me.